### DISCUSSION TITLE
希望 Mac版 也可以导出图片

### DISCUSSION THREAD

**Discussion by Coldfengzi** [tamlok/vnote#1920] [Ideas]
目前Mac版只可以借助第三方工具导成pdf，而Windows版无需第三方工具就可以导出png或jpg，希望Mac版也有此功能，借助第三方也行

**Comment by tamlok** [comment]
mac应该也可以呀？怎么不行了？截图或者详细说一下。

**Reply by Coldfengzi** [reply]
![image](https://user-images.githubusercontent.com/23092924/140874666-701de863-eaab-4449-ae43-c632dfeef0ee.png)

**Reply by tamlok** [reply]
然后这里按导出，是不行吗？Windows也这几个选项呀。要导出图片的话，得下载wkhtml2pdf来搞。

**Reply by Coldfengzi** [reply]
> 然后这里按导出，是不行吗？Windows也这几个选项呀。要导出图片的话，得下载wkhtml2pdf来搞。

谢谢，用wkhtmltoimage   -f jpeg %1 %5 搞定了

**Comment by tamlok** [comment]
win怎么可以内置导出png?
